docs/develop-resources/README.md

```diff
@@ -18,8 +22,14 @@ There are definitely more work need to be done, but we'd like to share the steps
 
 ## Develop by scenraios
 
+You may only need to add a new field, or to promote an Alpha resource to Beta rather than writing an entirely new resource. To figure out the most suitable scenario, you need to know the resources current status:
+- [ ] Check the latest [CRD](https://github.com/GoogleCloudPlatform/k8s-config-connector/tree/master/crds). If the resource exists but in Alpha, you won’t see it in  [Config Connector API Reference](https://cloud.google.com/config-connector/docs/reference/overview) until it is promoted to Beta. Please follow [promote alpha to beta](./scenarios/alpha-to-beta.md) guide.
+- [ ] If no CRD is found,  you do need to add a new resource, please file an issue first to let us know. We can help make sure the resource is not already taken care of by someone else.
```

Comment:
you do need -> you need

Reply:
Also better to break this into smaller sentences.

If no CRD is found, a need resource needs to be added. To begin please see if there is already an issue filed for this resource. If there is please comment on the issue to indicate increased interest in it. If you don't find an issue for the new resource, please start by filing the issue. If you are volunteering to work on the new resource, please assign the issue to yourself. This helps make sure the resource is not already taken care of by someone else.